ml_perf/README.md

```diff
@@ -0,0 +1,114 @@
+To setup the environment on Ubuntu 16.04 (16 CPUs, one P100, 100 GB disk), you can use these
+commands. This may vary on a different operating system or graphics card.
+
+```
+    # Clone repository
+    git clone https://github.com/tensorflow/minigo
+    # Note: This will eventually change to:
+    # git clone http://github.com/mlperf/training
+
+    # Install dependencies
+    apt-get install -y python3 python3-pip rsync git wget pkg-config zip g++ zlib1g-dev unzip
+
+    # Install python dependencies
+    pip3 install -r requirements.txt
+
+    # Install bazel
+    wget https://github.com/bazelbuild/bazel/releases/download/0.17.1/bazel-0.17.1-installer-linux-x86_64.sh
+    chmod +x bazel-0.17.1-installer-linux-x86_64.sh
+    ./bazel-0.17.1-installer-linux-x86_64.sh
+
+    # Compile TensorFlow
+    ./cc/configure_tensorflow.sh
+
+    # Compile and run C++ self-play and evaluation binaries
+    bazel build  -c opt  --define=tf=1  --define=board_size=9  cc:selfplay  cc:eval
+
+    # Download target model
+    gsutil -m cp gs://tensor-go-ml-perf/models/9x9/target.* ml_perf/
+
+    # Run training loop
+    BOARD_SIZE=9  python  ml_perf/reference_implementation.py --base_dir=$(pwd)/results/$(date +%Y-%m-%d)
+```
+
+### Steps to download and verify data
+Unlike other benchmarks, there is no data to download. All training data comes from games played
+during benchmarking.
+
+# 3. Model
+### Publication/Attribution
+
+This benchmark is based on the minigo project (https://github.com/tensorflow/minigo); which is
+inspired by the work done by Deepmind with ["Mastering the Game of Go with Deep Neural Networks and
+Tree Search"](https://www.nature.com/articles/nature16961), ["Mastering the Game of Go without Human
+Knowledge"](https://www.nature.com/articles/nature24270), and ["Mastering Chess and Shogi by
+Self-Play with a General Reinforcement Learning Algorithm"](https://arxiv.org/abs/1712.01815).
+Note that MiniGo is an independent effort from AlphaGo, and that this fork is minigo is independent
+from minigo itself.
+
+### Reinforcement Setup
+This benchmark includes both the environment and training for 9x9 Go. There are four primary phases
+in this benchmark, these phases are repeated in order:
+
+ - Selfplay: the *current best* model plays games against itself to produce board positions for
+   training.
+ - Training: train the neural networks using selfplay data from recent models. We are not using the
+   selfplay data from the current iteration so that selfplay and the rest of the iteration may run
+   in parallel.
+ - Model Evaluation: the *current best* and the most recently trained model play a series of games.
+   In order to become the new *current best*, the most recently trained model must win 55% of the
+   games.
+ - Target Evaluation: if the newly trained model has been promoted to the current best, play a series
+   of games against a previously trained model. The termination criteria for the benchmark is to win
```

Comment:
Specify what "previously trained model" is. Do you mean from earlier this run, from us, from gnugo...

Reply:
I'm not sue we've actually decided what the target model should be. I've put "a target model that was previously trained via this reference benchmark" for now, which what I believe is the simplest approach: selfplay against a previously trained Minigo model can be easily done in parallel, while playing against a different engine like Leela or GnuGo over GTP requires adding extra support for parallel playing of games over GTP (see the [GtpEvaluator](http://go/gh/chsigg/training/blob/19x19/reinforcement/minigo.patch) that Christian had written in his original version of the benchmark).